db/migrate/20210204144205_create_follows.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFollows < ActiveRecord::Migration[6.1]
```

Comment:
@idprietop if this is not related to this PR, please remove it, thank you!

Reply:
Ok, i already removed it.